lib/db/camel-case.js

```diff
@@ -0,0 +1,13 @@
+const _ = require('lodash')
+
+module.exports = function (data, snakeCases) {
+  return data.map(record => {
+    return snakeCases.reduce((record, key) => {
+      if (record[key]) {
+        record[_.camelCase(key)] = record[key]
+        delete record[key]
+      }
+      return record
+    }, record)
+  })
+}
```

Comment:
minor: in general i'm not a fan of mutating an object, but seems fine

Reply:
yeah, agreed, and thought about that during this.  I'll think on it some more.

Reply:
The only other pattern that comes to mind would requiring using something like `_.omit`:

```js
module.exports = function (data, snakeCases) {
  return data.map(record => {
    const transformedProps = _.chain(record)
      .pick(record, ...snakeCases)
      .mapKeys((value, key) => {
        return _.camelCase(key)
      })
      .value()

    return { ..._.omit(record, ...snakeCases), ...transformedProps }
  })
}
```

Avoids mutating the object, but `_.omit` is supposed to be very slow.  Other the other hand, the records are unlikely to have many properties, so it might not be an issue in this particular use case.  Thoughts?